patches/minecraft/net/minecraft/entity/EntityLivingBase.java.patch

```diff
@@ -66,9 +107,23 @@
 +            this.field_70180_af.func_187227_b(field_184634_g, event.areParticlesHidden());
 +            this.field_70180_af.func_187227_b(field_184633_f, event.getColor());
              this.func_82142_c(this.func_70644_a(MobEffects.field_76441_p));
++            if(!field_70170_p.field_72995_K) { NBTTagList nbttaglist = new NBTTagList();
```

Comment:
Oh god That's nasty

Reply:
I am sorry, I am new to this so I am having trouble seeing the problem.